src/pfe/portal/modules/utils/sharedFunctions.js

```diff
@@ -146,6 +130,21 @@ module.exports.forceRemove = async function forceRemove(path) {
   }
 }
 
+/** C:\helloThere -> /c/helloThere */
+function convertFromWindowsDriveLetter(absolutePath) {
+  if (!isWindowsAbsolutePath(absolutePath)) {
+    return absolutePath;
+  }
+  let linuxPath;
+  // Replace \ with /
+  linuxPath = convertBackSlashesToForwardSlashes(absolutePath);
+  const char0 = linuxPath.charAt(0);
+  // Strip first two characters
```

Comment:
could get rid of comments?

Reply:
removed and changed `absolutePath` to `windowsPath` for clarity